.github/workflows/trunk-upgrade.yaml

```diff
@@ -34,29 +34,10 @@ jobs:
           reviewers: "@masterpointio/masterpoint-internal"
           prefix: "chore: "
 
-      - name: Wait for checks to pass + Merge PR
+      - name: Merge PR automatically
         if: steps.trunk-upgrade.outputs.pull-request-number != ''
         env:
-          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+          GITHUB_TOKEN: ${{ steps.generate-token.outputs.token }}
           PR_NUMBER: ${{ steps.trunk-upgrade.outputs.pull-request-number }}
         run: |
-          echo "Waiting for required status checks to pass on PR #$PR_NUMBER..."
-          while true; do
-            CHECKS_JSON=$(gh pr checks "$PR_NUMBER" --required --json state,bucket)
-            echo "Current checks status: $CHECKS_JSON"
-
-            if echo "$CHECKS_JSON" | jq -e '.[] | select(.bucket=="fail")' > /dev/null; then
-              echo "One or more required checks have failed. Exiting..."
-              exit 1
-            fi
-
-            FAILED_OR_PENDING_CHECKS=$(echo "$CHECKS_JSON" | jq '[.[] | select(.state!="SUCCESS" or .bucket!="pass")] | length')
-            if [ "$FAILED_OR_PENDING_CHECKS" -eq 0 ]; then
-              echo "All required checks passed. Merging PR #$PR_NUMBER..."
-              gh pr merge "$PR_NUMBER" --squash --delete-branch --admin
-              break
-            else
-              echo "Some required checks are still running or pending. Retrying in 30s..."
-              sleep 30
-            fi
-          done
+          gh pr merge "$PR_NUMBER" --squash --auto --delete-branch
```

Comment:
I think @gberenice wanted to keep this... but we'll let her weigh in tomorrow. We can revert if needed...